src/backend/aspen/workflows/nextstrain_run/build_config.py

```diff
@@ -126,14 +119,14 @@ def update_subsampling_for_location(tree_build_level, subsampling):
 
 def update_subsampling_for_country(subsampling):
     # State and country aren't useful
-    del subsampling["state"]
-    del subsampling["country"]
+    if subsampling["state"] exist, then del subsampling["state"]        # fake code. I cannot python...
+    if subsampling["country"] exist, then del subsampling["country"]    # fake code
```

Comment:
be careful here -- this is almost real python! I'm going to start expecting unit tests with your PR's soon 😝

Reply:
hahahaha i was 50% ok with python, and after realizing i understood the yaml builder code invocation totally backward, i'm now 20% confident